app/views/CreateChannelView.tsx

```diff
@@ -32,38 +31,41 @@ const styles = StyleSheet.create({
 	list: {
 		width: '100%'
 	},
-	input: {
-		height: 54,
-		paddingHorizontal: 18,
-		fontSize: 17,
-		...sharedStyles.textRegular
-	},
 	switchContainer: {
-		height: 54,
+		minHeight: 54,
```

Comment:
Why min? Is it expected to grow?

Reply:
![Screen Shot 2022-08-02 at 19 10 35](https://user-images.githubusercontent.com/47038980/182482121-8a48cba1-ea0f-463e-90d8-91120b8ba34b.png)


In this case, I added minHeight and MaxHeight because of the different descriptions

Reply:
I see.
I added a comment on this review about this usage.
On your screenshot, you can see that, although working as you expect, the layout has no padding for `broadcast`.
Instead of heights, you should be using padding.

Also broadcast description is pt-BR, while the rest is en.

Reply:
The translation was fixed :) About the padding, you are right! I will change it